src/app/modules/main/chat_section/controller.nim

```diff
@@ -177,7 +177,7 @@ proc init*(self: Controller) =
     self.events.on(SIGNAL_COMMUNITY_CHANNEL_CATEGORY_CHANGED) do(e:Args):
       let args = CommunityChatOrderArgs(e)
       if (args.communityId == self.sectionId):
-        self.delegate.onCommunityCategoryChannelChanged(args.chatId, args.categoryId)
```

Comment:
`onCommunityCategoryChannelChanged` just causes the chat in question to be set as active (which I don't even think is desired), so this can be safely removed here.